common-lib/src/main/scala/com/gu/mediaservice/lib/config/CommonConfigWithElastic.scala

```diff
@@ -17,8 +17,6 @@ class CommonConfigWithElastic(resources: GridConfigResources) extends CommonConf
   val persistenceIdentifier = string("persistence.identifier")
   val queriableIdentifiers = Seq(persistenceIdentifier)
 
-  val persistedRootCollections: List[String] = stringOpt("persistence.collections") match {
-    case Some(collections) => collections.split(',').toList
-    case None => List(s"${staffPhotographerOrganisation} Archive")
-  }
+  val maybePersistOnlyTheseCollections: Option[List[String]] = stringOpt("persistence.onlyTheseCollections")
+    .map(_.split(',').toList)
```

Comment:
understandable as the previous config modelled this as string with comma separated values too, but maybe we could take this opportunity of renaming the key to model properly as a list of strings instead? if only cause then you can choose to persist a collection that's named with a comma (weird but afaik a legit choice)

Reply:
great shout, doing that now

Reply:
done, with (existing, albeit refactored) fallback to interpret comma based too